trust/meson.build

```diff
@@ -103,6 +103,11 @@ executable('trust',
                           libp11_tool_dep] + libffi_deps + dlopen_deps + libtasn1_deps,
            install: true)
 
+executable('trust-check-format',
```

Comment:
Is it possible to merge this into the `trust` command rather than a separate executable?